app/Topic.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Topic extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name', 'description', 'subject_id',
+    ];
```

Comment:
Add BelongsTo relation for subject.
Also, add HasMany relation in Subject model for topics.